src/controller/playRace.js

```diff
@@ -0,0 +1,33 @@
+import { CAR_CAN_GO_COUNT } from '../util/constants.js';
+import generateCars from '../model/generateCars.js';
+import showResult from '../view/showResult.js';
+import getWinners from '../model/getWinners.js';
+import showWinners from '../view/showWinners.js';
+import showRestart from '../view/showRestart.js';
+
+function sortCars(cars) {
+  const sortedCars = [...cars];
+  return sortedCars.sort((a, b) => b.position - a.position);
+}
```

Comment:
const sortedCars = [...cars] // 요 라인부터 정렬이 되지 않았기 때문에... 바로 리턴하는 건 어떨까 싶습니다.  
```suggestion
function sortCars(cars) {
  return [...cars].sort((a, b) => b.position - a.position);
}
```